webapp/src/ts/components/filters/analytics-filter/analytics-filter.component.html

```diff
@@ -1,5 +1,7 @@
 <div class="filters tab-bar">
-
+  <button class="app-menu-button" mat-icon-button (click)="openMenu()" aria-label="Application menu">
```

Comment:
@Benmuiruri I added the button to open the sidebar menu here temporarily to unblock @kapelo 

I couldn't add it to the analytics component (like reports, contacts, etc.) because it would push the other elements of this bar out of the viewport.  

When you work on https://github.com/medic/care-teams/issues/130, please fix this button so it is in the same place as the other pages That colored-bar could be a reusable component. For example:

```
<topbar> // adds button to open menu and navigation component
   <filters>
   <filters>
</topbar> 
```
The topbar's html automatically adds this button and the <navigation> component. And then displays the filters that are injected in the middle.